lib/utilities/utility.js

```diff
@@ -72,16 +72,22 @@ function annotateChildPods(kubectl, resourceType, resourceName, allPods) {
 }
 exports.annotateChildPods = annotateChildPods;
 function annotateNamespace(kubectl, namespaceName) {
-    let annotate = true;
     const result = kubectl.getResource('namespace', namespaceName);
-    this.checkForErrors([result]);
-    const annotationsSet = JSON.parse(result.stdout).metadata.annotations;
-    if (!!annotationsSet && !!annotationsSet.runUri && annotationsSet.runUri.indexOf(process.env['GITHUB_REPOSITORY']) == -1) {
-        annotate = false;
-        core.debug(`Skipping 'annotate namespace' as namespace annotated by other workflow`);

```

Comment:
Why are we skipping here btw?

What if there are 2 repos targeting the same namespace? Would we only show traceability for one pipeline?

Reply:
Yes, as per mvp, for differentiating two workflows we were taking an assumption of two different repos. In case of two workflows we will skip annotate namespace when we find repo annotations don't match. But I am working on adding a new condition that will look for the combination of repo + username + workflow two differentiate the workflows and we will not be skipping annotation at this time but actually append it in existing annotations as a json array.